.golangci.yaml

```diff
@@ -40,7 +40,7 @@ linters-settings:
       - name: blank-imports
       - name: context-as-argument
       - name: context-keys-type
-      - name: dot-imports
+      # - name: dot-imports
```

Comment:
Why don't you remove these commented lines?

Reply:
Because these linters are good and we do want to enable them. I have fixes for most of them but the PR became too big with all the fixes at once.

In this PR, I disable them by commenting out those lines and in the subsequent PRs, we can enable them one by one.